docs/FAQ.md

```diff
@@ -72,9 +124,9 @@ This CRD captures the status of the execution of a Plan defined in a `FrameworkV
 
 Parameters provide configuration for the Instance.
 
-#### What is a Strategy?
+#### What is a Deployment Strategy?
 
-The following strategies are available by default, these can be used in a `FrameworkVersion` YAML definition:
+A deployments strategy indicates the way in which a plan, or step must be executed.  If a step requires another step to complete first it is necessary to declare them as `serial`.  The following strategies are available by default, these can be used in a `FrameworkVersion` YAML definition:
```

Comment:
A deployment~~s~~ strategy...?
whitespaces after `executed.    If` and `serial.   The following`
We also should agree on if we use `step` or `Step` probably.

Reply:
there are different strategies... the issue here is this original said `What is a strategy`... then defined a deployment strategy (consist with SDK deployment strategies)... but then in `What is a Trigger` it talks about "update strategy".   It seems like a mistake to define "strategies" in isolation.     I'm fixing all the whitespace.